cdap-data-fabric-tests/src/test/java/co/cask/cdap/data2/dataset2/lib/table/hbase/HBaseMetricsTableTest.java

```diff
@@ -86,6 +96,36 @@ public static void tearDown() throws Exception {
   }
 
   @Override
+  @Test
+  public void testConcurrentIncrement() throws Exception {
+    final MetricsTable table = getTable("testConcurrentIncrement");
+    final int rounds = 500;
+    Map<byte[], Long> inc1 = ImmutableMap.of(X, 1L, Y, 2L);
+    Map<byte[], Long> inc2 = ImmutableMap.of(Y, 1L, Z, 2L);
+    // HTable used by HBaseMetricsTable is not thread safe, so each thread must use a separate instance
```

Comment:
Shouldn't we instead modify MetricsTableTest? Since we assume MetricsTable is not thread-safe